### PR TITLE
Create ip-masq-agent.md

### PR DESCRIPTION
**Pull request by jiaj12**
zh-trans content/en/docs/tasks/administer-cluster/ip-masq-agent.md

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/ip-masq-agent.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
+---
+title: IP Masquerade Agent User Guide
```
Comment (sftim) [inline content/zh/docs/tasks/administer-cluster/ip-masq-agent.md]: This title is untranslated.

File: content/zh/docs/tasks/administer-cluster/ip-masq-agent.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+{{% capture discussion %}}
+<!--
+## IP Masquerade Agent User Guide
+-->
+## IP Masquerade Agent 用户指南
+
+<!--
+The ip-masq-agent configures iptables rules to hide a pod's IP address behind the cluster node's IP address. This is typically done when sending traffic to destinations outside the cluster's pod [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) range.
+-->
+ip-masq-agent 配置 iptables 规则以隐藏位于集群节点 IP 地址后面的 pod 的 IP 地址。 这通常在将流量发送到集群的 pod [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) 范围之外的目的地时使用。
```
Comment (sftim) [inline content/zh/docs/tasks/administer-cluster/ip-masq-agent.md]: This links to a page in English-language Wikipedia. Is that intentional?

File: content/zh/docs/tasks/administer-cluster/ip-masq-agent.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+*   **Masquerading**
+    A form of NAT that is typically used to perform a many to one address translation, where multiple source IP addresses are masked behind a single address, which is typically the device doing the IP routing. In Kubernetes this is the Node's IP address.
+-->    
+*   **伪装**
+    NAT 的一种形式，通常用于执行多对一地址转换，其中多个源 IP 地址被隐藏在单个地址后面，该地址通常是执行 IP 路由的设备。在 Kubernetes 中，这是节点的 IP 地址。
+<!--
+*   **CIDR (Classless Inter-Domain Routing)**
+    Based on the variable-length subnet masking, allows specifying arbitrary-length prefixes. CIDR introduced a new method of representation for IP addresses, now commonly known as **CIDR notation**, in which an address or routing prefix is written with a suffix indicating the number of bits of the prefix, such as 192.168.2.0/24.
+-->
+*   **CIDR (无类别域内路由)**
```
Comment (sftim) [inline content/zh/docs/tasks/administer-cluster/ip-masq-agent.md]:
```suggestion
*   **CIDR (无类别域间路由)**
```

File: content/zh/docs/tasks/administer-cluster/ip-masq-agent.md [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+-->
+ip-masq-agent 配置 iptables 规则，以便在将流量发送到集群节点的IP和集群IP范围之外的目标时处理伪装节点/pod 的 IP 地址。这基本上隐藏了集群节点 IP 地址后面的pod IP地址。在某些环境中，去往“外部”地址的流量必须从已知的机器地址发出。例如，在 Google Cloud 中，任何到互联网的流量都必须来自 VM 的 IP。使用容器时，如 Google Kubernetes Engine，从Pod IP 发出的流量将被拒绝出出站。为了避免这种情况，我们必须将 Pod IP 隐藏在 VM 自己的IP地址后面 - 通常称为“伪装”。默认情况下，代理配置为将[RFC 1918](https://tools.ietf.org/html/rfc1918)指定的三个私有IP范围视为非伪装[CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing)。这些范围是 10.0.0.0/8,172.16.0.0/12 和 192.168.0.0/16。默认情况下，代理还将链路本地地址(169.254.0.0/16)视为非伪装 CIDR。代理程序配置为每隔60秒从*/etc/config/ip-masq-agent*重新加载其配置，这也是可修改的。
```
Comment (sftim) [inline content/zh/docs/tasks/administer-cluster/ip-masq-agent.md]: This links to a page in English-language Wikipedia. Is that intentional?

File: content/zh/docs/tasks/administer-cluster/ip-masq-agent.md [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+
+![masq/non-masq example](/images/docs/ip-masq.png)
```
Comment (sftim) [inline content/zh/docs/tasks/administer-cluster/ip-masq-agent.md]: This alternate text (for accessibility, etc) is not localized.

File: content/zh/docs/tasks/administer-cluster/ip-masq-agent.md [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+
+<!--
+The agent configuration file must be written in YAML or JSON syntax, and may contain three optional keys:
+-->
+代理配置文件必须使用 YAML 或 JSON 语法编写，并且可能包含三个可选值：
+
+<!--
+*   **nonMasqueradeCIDRs:** A list of strings in [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) notation that specify the non-masquerade ranges.
+-->
+*   **nonMasqueradeCIDRs:** [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) 表示法中的字符串列表，用于指定不需伪装的地址范围。
```
Comment (sftim) [inline content/zh/docs/tasks/administer-cluster/ip-masq-agent.md]: This links to a page in English-language Wikipedia. Is that intentional?

File: content/zh/docs/tasks/administer-cluster/ip-masq-agent.md (hunk 2 above)
Comment (jiaj12) [reply content/zh/docs/tasks/administer-cluster/ip-masq-agent.md]: https://zh.wikipedia.org/wiki/%E6%97%A0%E7%B1%BB%E5%88%AB%E5%9F%9F%E9%97%B4%E8%B7%AF%E7%94%B1
here is the corresponding zh page, shall I replace it ?

Comment (sftim) [reply content/zh/docs/tasks/administer-cluster/ip-masq-agent.md]: Sounds good to me @jiaj12

Comment (jiaj12) [reply content/zh/docs/tasks/administer-cluster/ip-masq-agent.md]: all done

File: content/zh/docs/tasks/administer-cluster/ip-masq-agent.md (hunk 5 above)
Comment (jiaj12) [reply content/zh/docs/tasks/administer-cluster/ip-masq-agent.md]: i'm not sure which part are you refer to, can you be more specific

File: content/zh/docs/tasks/administer-cluster/ip-masq-agent.md [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+target     prot opt source               destination
+RETURN     all  --  anywhere             169.254.0.0/16       /* ip-masq-agent: 集群本地流量不被 MASQUERADE 控制 */ ADDRTYPE match dst-type !LOCAL
+RETURN     all  --  anywhere             10.0.0.0/8           /* ip-masq-agent: cluster-local
+MASQUERADE  all  --  anywhere             anywhere             /* ip-masq-agent: 出站流量应受 MASQUERADE 控制 (此规则必须在集群本地 CIDR 规则之后) */ ADDRTYPE match dst-type !LOCAL
+```
+
+<!--
+By default, the link local range (169.254.0.0/16) is also handled by the ip-masq agent, which sets up the appropriate iptables rules.  To have the ip-masq-agent ignore link local, you can set *masqLinkLocal*  to true in the config map.
+-->
+默认情况下，本地链路范围 (169.254.0.0/16) 也由 ip-masq agent 处理，该代理设置适当的 iptables 规则。 要使 ip-masq-agent 忽略本地链路，可以在配置映射中将 *masqLinkLocal*设置为true。
```
Comment (xichengliudui) [inline content/zh/docs/tasks/administer-cluster/ip-masq-agent.md]:
```suggestion
默认情况下，本地链路范围 (169.254.0.0/16) 也由 ip-masq agent 处理，该代理设置适当的 iptables 规则。 要使 ip-masq-agent 忽略本地链路，可以在配置映射中将 *masqLinkLocal* 设置为true。
```

File: content/zh/docs/tasks/administer-cluster/ip-masq-agent.md [hunk 8]
```diff
@@ -0,0 +1,215 @@
...
+
+<!--
+More information can be found in the ip-masq-agent documentation [here](https://github.com/kubernetes-incubator/ip-masq-agent)
+-->
+更多信息可以通过 ip-masq-agent 文档 [这里](https://github.com/kubernetes-incubator/ip-masq-agent) 找到
+
+<!--
+In most cases, the default set of rules should be sufficient; however, if this is not the case for your cluster, you can create and apply a [ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/) to customize the IP ranges that are affected.  For example, to allow only 10.0.0.0/8 to be considered by the ip-masq-agent, you can create the following [ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/) in a file called "config".
+-->
+在大多数情况下，默认的规则集应该足够；但是，如果您的群集不是这种情况，则可以创建并应用[ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/) 来自定义受影响的 IP 范围。 例如，要允许 ip-masq-agent 仅作用于10.0.0.0/8，您可以一个名为 “config” 的文件中创建以下[ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/) 。
```
Comment (xichengliudui) [inline content/zh/docs/tasks/administer-cluster/ip-masq-agent.md]:
```suggestion
在大多数情况下，默认的规则集应该足够；但是，如果您的群集不是这种情况，则可以创建并应用 [ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/) 来自定义受影响的 IP 范围。 例如，要允许 ip-masq-agent 仅作用于 10.0.0.0/8，您可以一个名为 “config” 的文件中创建以下 [ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/) 。
```

File: content/zh/docs/tasks/administer-cluster/ip-masq-agent.md [hunk 9]
```diff
@@ -0,0 +1,215 @@
...
+您必须同时将适当的节点标签应用于集群中希望代理运行的任何节点。
+
+`
+kubectl label nodes my-node beta.kubernetes.io/masq-agent-ds-ready=true
+`
+
+<!--
+More information can be found in the ip-masq-agent documentation [here](https://github.com/kubernetes-incubator/ip-masq-agent)
+-->
+更多信息可以通过 ip-masq-agent 文档 [这里](https://github.com/kubernetes-incubator/ip-masq-agent) 找到
```
Comment (xichengliudui) [inline content/zh/docs/tasks/administer-cluster/ip-masq-agent.md]:
```suggestion
更多信息可以通过[这里](https://github.com/kubernetes-incubator/ip-masq-agent) 找到  ip-masq-agent 文档。
```

**Pull request merged**